src/Utilities.UnitTests/ToolTask_Tests.cs

```diff
@@ -155,11 +151,8 @@ public void Regress_Mutation_WarnIfCommandLineTooLong()
 
                 // "cmd.exe" croaks big-time when given a very long command-line.  It pops up a message box on
                 // Windows XP.  We can't have that!  So use "attrib.exe" for this exercise instead.
-#if FEATURE_SPECIAL_FOLDERS
-                t.FullToolName = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.System), NativeMethodsShared.IsWindows ? "attrib.exe" : "ps");
-#else
-                t.FullToolName = Path.Combine(FileUtilities.GetFolderPath(FileUtilities.SpecialFolder.System), NativeMethodsShared.IsWindows ? "attrib.exe" : "ps");
-#endif
+                string systemPath = NativeMethodsShared.IsUnixLike ? "/bin" : Environment.GetFolderPath(Environment.SpecialFolder.System);
```

Comment:
Do we actually error on long command lines on non-Windows? We probably shouldn't. But that can wait.